skipscale/config.py

```diff
@@ -100,10 +100,10 @@ def sentry_dsn(self) -> Optional[str]:
             return self.validated_config["sentry_dsn"]
         return None
 
-    def sentry_traces_sample_rate(self) -> Optional[float]:
+    def sentry_traces_sample_rate(self) -> float:
         if "sentry_traces_sample_rate" in self.validated_config:
             return self.validated_config["sentry_traces_sample_rate"]
-        return None
+        return 1.0
 
```

Comment:
The intent here was for tracing to be disabled unless `sentry_traces_sample_rate` was specified in the config.

Reply:
typing sez that the sentry-sdk parameter is not Optional, changed to 0.0.